### ISSUE TITLE
Changing the storage path doesn't take affect till the app is restarted

### ISSUE THREAD

**Issue by aza547** [aza547/wow-recorder#158]
```
 might work on that at some point when this annoys me enough 😄
Aha, I know what the issue is.
OBS basic.ini:

RecFilePath=Z:\Videos\.temp\

even when i changed it. Maybe this file only gets written when it first starts up.
```

To reproduce:
- Start the app
- Change the config such that the storage path is moved
- Run a test
- Test records to old location

Workaround:
- Restart the application

**Comment by aza547** [comment]
I thought I saw this NOT happen when reproducing with the debugger attached. Had a theory that some timing condition. Might be the same as #141

**Comment by aza547** [comment]
This is fixed in 3.3.0 with the OBS improvements there.
